docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -452,12 +459,12 @@ Flush metadata cache
 
 * ``system.flush_metadata_cache()``
 
-  Flush all metadata caches.
+  Flushes all metadata caches.
 
 * ``system.flush_metadata_cache(schema_name => ..., table_name => ...)``
 
-  Flush metadata caches entries connected with selected table.
-  Procedure requires named parameters to be passed
+  Flushes metadata cache entries connected with selected table.
```

Comment:
```suggestion
  Flushes metadata cache entries connected with a specific table.
```

or 

```suggestion
  Flushes metadata cache entries of a specific table.
```

Reply:
Went with 2nd suggestion. Would just commit suggestion but would rather not rely on gui for Trino fork.